travel_platform/api/foursquare/foursquare_data.py

```diff
@@ -0,0 +1,36 @@
+import json
+import foursquare
```

Comment:
json was used for testing. Already removed.
foursquare - python wrapper for the Foursquare API.